app/src/main/java/com/concordia/mcga/fragments/TransportButtonFragment.java

```diff
@@ -15,23 +17,39 @@
 
 public class TransportButtonFragment extends Fragment implements View.OnClickListener {
 
+    /**
+     * Enum representing which transport type is selected
+     */
+    enum TransportType {
```

Comment:
make this into a class, with static types. Own domain model already has Transportation as a model and I think this is covering the same information. Also, add the icons as an attribute to that class.

Something along the lines of:

public class Transportation {
      enum TransportType {...}
      TransportType type
      int iconID

static WALK = new Transportation();
}

then you can use Transportation in this fragment.